Packs/XMCO/Integrations/Serenety/Serenety.yml

```
@@ -0,0 +1,72 @@
+fromversion: 6.10.0
+category: Vulnerability Management
+sectionOrder:
+  - Connect
+  - Collect
+commonfields:
+  id: Serenety
+  version: -1
+configuration:
+- display: XMCO LePortail instance URL
+  defaultvalue: https://leportail.xmco.fr/
+  name: url
+  required: true
+  type: 0
+  section: Connect
+- displaypassword: API Key
+  name: api_key
+  required: true
+  type: 9
+  hiddenusername: true
+  section: Connect
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+  section: Connect
+  advanced: true
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+  section: Connect
+  advanced: true
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  section: Collect
+- display: Maximum number of incidents per fetch
+  defaultvalue: '50'
+  name: max_fetch
+  type: 0
+  required: false
+  section: Collect
+- defaultvalue: 3 days
+  display: First fetch time
+  additionalinfo: First alert created date to fetch.
+  name: first_fetch
+  type: 0
+  required: false
+  section: Collect
+- display: Incident type
+  name: incidentType
```

Add here a key for a default value

```suggestion
  name: incidentType
  defaultvalue: 'XMCO Serenety Alert'
```